api/v1/app.py

```diff
@@ -1,31 +1,38 @@
 #!/usr/bin/python3
-# App for the api
-from flask import Flask, make_response, jsonify
+"""
+flask app module
+"""
+
+
+import os
 from models import storage
```

Comment:
user imports should be at bottom, see https://www.python.org/dev/peps/pep-0008/#imports for more details